src/plugins/data/common/search/expressions/esaggs/esaggs_fn.ts

```diff
@@ -118,9 +119,13 @@ export async function handleEsaggsRequest(
   args: Arguments,
   params: RequestHandlerParams
 ): Promise<Datatable> {
-  const resolvedTimeRange = input?.timeRange && calculateBounds(input.timeRange);
+  const resolvedTimeRange =
```

Comment:
nit: 'input' argument is no longer used

Reply:
no longer relevant with https://github.com/elastic/kibana/pull/84405/commits/d001f1b1d9868f6b5f0ab8aba0b183b57e7601db change